conda_smithy/lint_recipe.py

```diff
@@ -96,34 +99,51 @@ def lintify_forge_yaml(recipe_dir=None) -> (list, list):
 
 
 def lintify_meta_yaml(
-    meta, recipe_dir=None, conda_forge=False
+    meta,
+    recipe_dir=None,
+    conda_forge=False,
+    is_rattler_build=False,
 ) -> (list, list):
     lints = []
     hints = []
     major_sections = list(meta.keys())
 
     # If the recipe_dir exists (no guarantee within this function) , we can
     # find the meta.yaml within it.
-    meta_fname = os.path.join(recipe_dir or "", "meta.yaml")
+    recipe_name = "meta.yaml" if not is_rattler_build else "recipe.yaml"
+    recipe_fname = os.path.join(recipe_dir or "", recipe_name)
 
-    sources_section = get_section(meta, "source", lints)
-    build_section = get_section(meta, "build", lints)
-    requirements_section = get_section(meta, "requirements", lints)
+    sources_section = get_section(meta, "source", lints, is_rattler_build)
+    build_section = get_section(meta, "build", lints, is_rattler_build)
+    requirements_section = get_section(
+        meta, "requirements", lints, is_rattler_build
+    )
     build_requirements = requirements_section.get("build", [])
     run_reqs = requirements_section.get("run", [])
-    test_section = get_section(meta, "test", lints)
-    about_section = get_section(meta, "about", lints)
-    extra_section = get_section(meta, "extra", lints)
-    package_section = get_section(meta, "package", lints)
-    outputs_section = get_section(meta, "outputs", lints)
+    if is_rattler_build:
+        test_section = get_section(meta, "tests", lints, is_rattler_build)
+    else:
+        test_section = get_section(meta, "test", lints, is_rattler_build)
+    about_section = get_section(meta, "about", lints, is_rattler_build)
+    extra_section = get_section(meta, "extra", lints, is_rattler_build)
+    package_section = get_section(meta, "package", lints, is_rattler_build)
+    outputs_section = get_section(meta, "outputs", lints, is_rattler_build)
```

Comment:
The API of `get_section` hasn't changed as there is a default value for the fourth argument (false).